.gitignore

```
@@ -28,7 +28,10 @@ __pycache__/
 analysis_templates/*.html
 analysis_templates/*.html.zip
 
-# celltype references 
+# celltype references and data 
 celltype_annotation/references*
+celltype_annotation/data*
```

Stars should not be needed here, and may cause trouble
```suggestion
celltype_annotation/references
celltype_annotation/data
```